lib/appsignal/rack/rails_instrumentation.rb

```diff
@@ -6,59 +6,27 @@ module Appsignal
   # @api private
   module Rack
     class RailsInstrumentation
```

Comment:
I'm not quite sure what this is doing now, how this would be used, or how it sets errors.

Reply:
It is in progress - like I said, it's a sketch for now (because so many things need to get touched)